src/Microsoft.Azure.WebJobs.Host/Listeners/HostListenerFactory.cs

```diff
@@ -14,13 +14,15 @@
 using Microsoft.Azure.WebJobs.Logging;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
 
 namespace Microsoft.Azure.WebJobs.Host.Listeners
 {
     internal class HostListenerFactory : IListenerFactory
     {
         private static readonly MethodInfo JobActivatorCreateMethod = typeof(IJobActivator).GetMethod("CreateInstance", BindingFlags.Public | BindingFlags.Instance).GetGenericMethodDefinition();
         private const string IsDisabledFunctionName = "IsDisabled";
+        private const string TargetBaseScalingEnabled = "TARGET_BASED_SCALING_ENABLED";
```

Comment:
Let's add a constant for this and use it in the source/tests

Reply:
This check was moved to function host